index.js

```diff
@@ -208,8 +234,11 @@ FocusGroup.prototype.addMember = function(memberData, index) {
 
   var cleanedNodeText = nodeText.replace(/[\W_]/g, '').toLowerCase();
   var member = {
+    id: memberData.id,
+    index: index,
     node: node,
     text: cleanedNodeText,
+    value: memberData.value
```

Comment:
I was thinking about doing that, so yes! 😄  React Motion does that and it seems to work really well.